.github/workflows/ci_workflows.yml

```
@@ -0,0 +1,123 @@
+name: CI
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+
+          - name: Code style checks
+            os: ubuntu-latest
+            python: 3.x
+            toxenv: codestyle
+
+          - name: Python 3.7 with minimal dependencies
+            os: ubuntu-latest
+            python: 3.7
+            toxenv: py37-test
+
+          - name: Python 3.8 with all optional dependencies
+            os: ubuntu-latest
+            python: 3.8
+            toxenv: py38-test-alldeps
+            toxargs: -v --develop
+            toxposargs: --open-files
+
+          # NOTE: In the build below we also check that tests do not open and
+          # leave open any files. This has a performance impact on running the
+          # tests, hence why it is not enabled by default.
+          - name: Python 3.7 with oldest supported version of all dependencies
+            os: ubuntu-16.04
+            python: 3.7
+            toxenv: py37-test-oldestdeps
+
+          - name: Python 3.7 with numpy 1.17 and full coverage
+            os: ubuntu-latest
+            python: 3.7
+            toxenv: py37-test-alldeps-numpy117-cov
+            toxposargs: --remote-data=astropy
+
+          - name: Python 3.8 with all optional dependencies (Windows)
+            os: windows-latest
+            python: 3.8
+            toxenv: py38-test-alldeps
+            toxposargs: --durations=50
+
+          - name: Python 3.7 with all optional dependencies (MacOS X)
+            os: macos-latest
+            python: 3.7
+            toxenv: py37-test-alldeps
+            toxposargs: --durations=50
+
+          - name: Python 3.6 with astropy LTS
+            os: ubuntu-16.04
+            python: 3.6
+            toxenv: py36-test-astropylts-numpy116
+            toxposargs: --remote-data=astropy
+
+          - name: Python 3.7 doc build
+            os: ubuntu-latest
+            python: 3.7
+            toxenv: build_docs
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Install language-pack-de and tzdata
+      if: startsWith(matrix.os, 'ubuntu')
+      run: sudo apt-get install language-pack-de tzdata
```

```suggestion
      run: sudo apt-get install language-pack-de tzdata graphviz
```
I think this is what's necessary to get the doc build to run...